### DISCUSSION TITLE
chia full node not connected error [BUG]

### DISCUSSION THREAD

**Discussion by Rekternino** [Chia-Network/chia-blockchain#9175] [Support]
I updated to the new version today and at first I couldn't login. When I logged in later, synchronization did not open in any way and remained in the not connected text. Still no change after waiting 12 hours.

When I deleted the blockchain db file and restored the old version, the problem was solved, but it started doing all the synchronization from scratch. I think this will take longer than 1 week. The db file is 30tb in size. I backed up the db file, but when I threw it again, I was stuck on again not connected text. Is there anyway to solve this problem.

**Comment by sargonas** [comment]
This could be one of many issues, moving this over to support where folks can try to help you out.

A few things that will play into this, to help get that conversation started:
- What version were you running before the update?
- What OS are you running, and are you running it in GUI or CLI mode?
